web/src/condense.js

```diff
@@ -197,6 +197,8 @@ export function show_message_condenser($row) {
 export function condense_and_collapse(elems) {
     const height_cutoff = message_viewport.height() * 0.65;
 
+    const rows_to_resize = [];
```

Comment:
I don't think you need the blank line above this.

Reply:
Removed